doc/conf.py

```diff
@@ -299,6 +300,7 @@ def setup(app):
     'matplotlib': ('http://matplotlib.org/', None),
 }
 
+from sphinx_gallery.sorting import ExplicitOrderStrict
```

Comment:
What benefit do we get from calling it `Strict` at the end? Why not just call it `ExplicitOrder` and then upon initialization give people the option of passing `strict=True` and defining what this does in the docstring?